terraform/environments/nomis-combined-reporting/locals_test.tf

```diff
@@ -53,13 +54,40 @@ locals {
           vpc_zone_identifier = module.environment.subnets["private"].ids
         }
         # autoscaling_schedules = module.baseline_presets.ec2_autoscaling_schedules.working_hours
+        CMS = {
+        random = {
+          length  = 11
+          special = false
+        }
+        description = "CMS password for connection to BI Platform"
+        }
```

Comment:
This needs to be inside the ssm_parameters option

Reply:
Thanks for catching that